data-prepper-core/src/main/java/com/amazon/dataprepper/DataPrepperExecute.java

```diff
@@ -0,0 +1,21 @@
+package com.amazon.dataprepper;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Execute entry into Data Prepper.
+ */
+public class DataPrepperExecute {
+    private static final Logger LOG = LoggerFactory.getLogger(DataPrepperExecute.class);
+
+    public static void main(String[] args) {
+        final DataPrepper dataPrepper = DataPrepper.getInstance();
+        if (args.length > 0) {
+            dataPrepper.execute(args[0]);
```

Comment:
Why don't we pass all the parameters?

Reply:
Good point - When we added this we were only looking at having the config file name - but yeah I agree, we should have passed all args. This is out of scope of this PR so I would take it up as a separate issue and fix it in separate PR. @erosas  u ok with it ?